common/src/main/java/tachyon/LeaderInquireClient.java

```diff
@@ -64,7 +64,8 @@ private LeaderInquireClient(String zookeeperAddress, String leaderPath) {
   public synchronized String getMasterAddress() {
     int tried = 0;
     try {
-      while (tried < MAX_TRY) {
+      int maxTry = new TachyonConf().getInt(Constants.ZOOKEEPER_LEADER_RETRY_COUNT);
```

Comment:
Can this be retrieved once per instance, instead of once per method call?

Also, this should probably use ClientContext.getConf()